pkg/glusterfs/controllerserver.go

```diff
@@ -0,0 +1,394 @@
+package glusterfs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gluster/gluster-csi-driver/pkg/glusterfs/utils"
+
+	csi "github.com/container-storage-interface/spec/lib/go/csi/v0"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/golang/glog"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	glusterDescAnn            = "GlusterFS-CSI"
+	glusterDescAnnValue       = "gluster.org/glusterfs-csi"
```

Comment:
why is this different from the driver definition of org.gluster.glusterfs? this is just a curious question, not really a ask for change.

Reply:
This `glusterDescAnn` and `glusterDescAnnValue` is a key value pair for identifying that the volume is created by CSI driver or not, IMO this need not be the `org.gluster.glusterfs`

Reply:
agrred to  @Madhu-1 .